presto-verifier/src/main/java/com/facebook/presto/verifier/rewrite/FunctionCallRewriter.java

```diff
@@ -144,20 +142,10 @@ public Node rewrite(Node root)
         return rewritten;
     }
 
-    public String getProcessedFunctionCallSubstitutes()
-    {
-        return processedFunctionCallSubstitutes.stream().map(functionCallSubstitute -> {
-                    String formattedOriginal = ExpressionFormatter.formatExpression(functionCallSubstitute.get(0), Optional.empty());
-                    String formattedSubstitute = ExpressionFormatter.formatExpression(functionCallSubstitute.get(1), Optional.empty());
-                    return String.format("% is substituted with %s", formattedOriginal, formattedSubstitute);
-                }
-        ).collect(Collectors.joining(", "));
-    }
-
```

Comment:
I see there is one more unused method [validateFunctionCallSubstitutes](https://github.com/prestodb/presto/blob/master/presto-verifier/src/main/java/com/facebook/presto/verifier/rewrite/FunctionCallRewriter.java#L105) in this class, should we remove that as well?

Reply:
Since that method doesn't have a format string mismatch, it's not related to this PR. Overall there are a lot of unused methods scattered throughout presto. We should think about what we want to do about that, but that's for another PR.